st2actions/st2actions/runners/windows_script_runner.py

```diff
@@ -125,12 +133,15 @@ def run(self, action_parameters):
         self._log_action_completion(logger=LOG, result=output, status=status, exit_code=exit_code)
         return (status, output, None)
 
-    def _run_script(self, script_path):
+    def _run_script(self, script_path, arguments=""):
```

Comment:
I would personally prefer to default this to `None` and below do something like:

```python
if arguments:
    command = 'powershell.exe %s %s' % (quote_windows(script_path), arguments)
else:
   command = 'powershell.exe %s' % (quote_windows(script_path))
```